extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -466,7 +466,7 @@ export class AmpAnalytics extends AMP.BaseElement {
     const threshold = parseFloat(spec['threshold']); // Threshold can be NaN.
     if (threshold >= 0 && threshold <= 100) {
       const key = this.expandTemplate_(spec['sampleOn'], trigger);
-      const keyPromise = urlReplacementsForDoc(this.win.document)
+      const keyPromise = urlReplacementsForDoc(this.getAmpDoc())
```

Comment:
Ditto: we need to figure out what's the right argument here. My general reaction is that it should be `this.element` most of the time, but please check.